aries-site/src/pages/templates/filtering.mdx

```diff
@@ -0,0 +1,81 @@
+import { Example } from '../../layouts';
+import {
+  FilteringCards,
+  FilteringLists,
+  FilteringWithRangeInput,
+  FilteringWithSelect,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/filtering/FilteringCards.js"
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <FilteringLists />
+</Example>
+
+## What is filtering?
+
+Filtering allows data to be narrowed down around a set of specified parameters. When users may not have a specific record in mind but are seeking to group data around a certain set of attributes, filtering should be used to allow the user to narrow the data and take action on relevant records.
+
+If a user will likely be looking for a specific record as opposed to groups of data, consider implementing [search](/templates/searching).
+
+## When to use
```

Comment:
```suggestion
## When to Use
```

Reply:
Wondering what convention we should take for capitalization -- I know that for the HPE Dev Newsletter we had switched to Sentence case to align with brand as opposed to title case. Thoughts on what is wanted here?

Reply:
I switched to sentence case on the table PR. I am okay with either as long as we are consistent going forward.